documentation/release-notes/topics/version_3.rst

```diff
@@ -104,6 +104,17 @@ Changes in version 3.9.2
 - **AS-12329** Fixed an issue that caused the Desired Rack Temperature warning threshold to be missing a "less than" condition
 - **AS-12392** Fixed an issue that caused the property list to be empty when adding new property-based sensor thresholds
 
+========================
+Changes in version 3.9.3
+========================
+- **AS-11880** Fixed an issue that caused certain Cisco switches to be merged
```

Comment:
Fixed an issue that caused certain network switches to be merged

Reply:
Thanks! I will address.

Reply:
Addressed